src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
```

Comment:
According to the checklist, class/interface names shouldn't be verbs. Consider renaming 'AreaCalculator' to something that represents an entity or a characteristic, like 'CalculableArea', to better reflect its purpose as an interface.